src/qcodes/instrument_drivers/Keithley/Keithley_2450.py

```diff
@@ -472,6 +473,24 @@ def __init__(self, parent: "Keithley2450", name: str, proper_function: str) -> N
             "when making a measurement.",
         )
 
+        self.add_parameter(
+            "block_during_ramp",
+            get_cmd=self._block,
+            set_cmd=self._set_block,
```

Comment:
If you replace the values for get_cmd and set_cmd with None here the parameter is a so called manual parameter that holds the state internally and you can avoid the _set_block attribute and simply use the  value directly